dist/browser/bootlint.js

```diff
@@ -11095,6 +11104,12 @@ var LocationIndex = _location.LocationIndex;
             reporter('Using `.pull-left` or `.pull-right` as part of the media object component is deprecated as of Bootstrap v3.3.0. Use `.media-left` or `.media-right` instead.', mediaPulls);
         }
     });
+    addLinter("W011", function lintMediaPulls($, reporter) {
+        var mediaPulls = $('.navbar .pull-left, .navbar .pull-right');
```

Comment:
```javascript
[
".navbar-nav.pull-right", 
".navbar-nav.pull-left", 
".navbar-text.pull-right", 
".navbar-text.pull-left", 
".navbar-btn.pull-right", 
".navbar-btn.pull-left", 
".navbar-form.pull-right", 
".navbar-form.pull-left", 
".navbar-link.pull-right", 
".navbar-link.pull-left"
]
```

This work? Obviously simplified with a forEach or map or something when building the selector...

Seems to fit with this:

> Align nav links, forms, buttons, or text, using the `.navbar-left` or `.navbar-right` utility classes. Both classes will add a CSS float in the specified direction. For example, to align nav links, put them in a separate `<ul>` with the respective utility class applied.